scripts/run-cppcheck.sh

```diff
@@ -8,7 +8,32 @@ AMICI_PATH=$(cd $SCRIPT_PATH/.. && pwd)
 
 cd ${AMICI_PATH}
 
-cppcheck -i${AMICI_PATH}/src/doc ${AMICI_PATH}/src 2> cppcheck.txt
+cppcheck -i${AMICI_PATH}/src/doc ${AMICI_PATH}/src  -I${AMICI_PATH}/include/ --enable=style 2> cppcheck.txt
+
+# suppress alloca warnings
+grep -v "(warning) Obsolete function 'alloca' called." cppcheck.txt > cppcheck_tmp.txt
+mv cppcheck_tmp.txt cppcheck.txt
+
+
+# suppress header warnings for standard libraries
+grep -v "Cppcheck cannot find all the include files" cppcheck.txt > cppcheck_tmp.txt
+mv cppcheck_tmp.txt cppcheck.txt
+
+grep -v "'AmiVectorArray' does not have a operator=" cppcheck.txt > cppcheck_tmp.txt
+mv cppcheck_tmp.txt cppcheck.txt
+
+grep -v "Member variable 'ExpData::nytrue_' is not initialized in the constructor" cppcheck.txt > cppcheck_tmp.txt
```

Comment:
Shouldn't we rather fix those? Either do in every constructor or set to zero in the class definition.

Reply:
this is a false positive, they are initialized in the delegate constructors which cppcheck does not recognize